FireboltNETSDK/Client/FireboltCommand.cs

```diff
@@ -311,7 +311,10 @@ public override int ExecuteNonQuery()
 
         public override object ExecuteScalar()
         {
-            throw new NotImplementedException();
+            using (DbDataReader reader = ExecuteReader())
+            {
+                return reader.Read() && reader.FieldCount > 0 ? reader.GetValue(0) : throw new InvalidOperationException("Cannot get scalar value from empty response");
```

Comment:
I believe we should return `Null` for an empty response.
From the docs: https://learn.microsoft.com/en-us/dotnet/api/system.data.common.dbcommand.executescalar?view=net-7.0#remarks
>If the first column of the first row in the result set is not found, a null reference (Nothing in Visual Basic) is returned. If the value in the database is null, the query returns DBNull.Value